Assets/Scripts/Utils/ClassExtender.cs

```diff
@@ -0,0 +1,15 @@
+﻿using UnityEngine;
+
+namespace Utils
+{
+    public static class ClassExtender
```

Comment:
Where is this even used?

Reply:
` if(!attackableLayers.Contains(other.gameObject.layer)) return;`
See targetFinder line 25